content/docs/user-guide/remote-content/use-a-remote-repository.md

```diff
@@ -0,0 +1,163 @@
+
+You can download templates from the **Create a New Project** page in the Project Manager by selecting the project template in the list of project templates and pressing the  **Download Template** button in the right pane.  
+
+
+## Using the `o3de` command line tool to access O3DE remote repositories 
+
+You can use the `o3de` command line tool to register, refresh, activate and deactivate remote repositories and download remote content.
+
+
+### Add a remote repository with the `o3de` command line tool
+
+You can use the `o3de` CLI tool to add O3DE remote repositories with the following command:
+
+{{< tabs name="Add an O3DE remote repository" >}}
+{{% tab name="Windows" %}}
+
+```cmd
+<engine>\scripts\o3de.bat register -ru <O3DE remote repository URI>
+```
+
+{{% /tab %}}
+{{% tab name="Linux" %}}
+
+```cmd
+<engine>/scripts/o3de.sh register -ru <O3DE remote repository URI>
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+See the [`o3de register` CLI Reference](/docs/user-guide/project-config/cli-reference/#register) for the complete list of options.
+
+### Activate or deactivate a remote repository with the `o3de` command line tool
+
+You can use the `o3de` CLI tool to activate or deactivate O3DE remote repositories with the following command:
+
+{{< tabs name="Activate an O3DE remote repository" >}}
+{{% tab name="Windows" %}}
+
+```cmd
+<engine>\scripts\o3de.bat repo --activate-repo <O3DE remote repository URI>
+```
+
+{{% /tab %}}
+{{% tab name="Linux" %}}
+
+```cmd
+<engine>/scripts/o3de.sh repo --activate-repo <O3DE remote repository URI>
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+When a remote repository is deactivated, you will no longer be able to see that remote repository's remote content which hasn't been downloaded in the Project Manager or download content from that remote repository until it is activated again.
+
+See the [`o3de repo` CLI Reference](/docs/user-guide/project-config/cli-reference/#repo) for the complete list of options.
+
+### Download content from remote repository with the `o3de` command line tool
+
+You can use the `o3de` CLI tool to download content from O3DE remote repositories with the following command:
+
+{{< tabs name="Download from an O3DE remote repository" >}}
+{{% tab name="Windows" %}}
+
+```cmd
+<engine>\scripts\o3de.bat download --gem-name <Gem name with optional version specifier>
+```
+
+Example which downloads a gem named `RemoteGem` version `1.0.0`:
+```cmd
+<engine>\scripts\o3de.bat download --gem-name RemoteGem==1.0.0
+```
+
+If no version specifier is given, the highest version available is downloaded.
+
+Example which downloads the latest version of a project named `RemoteProject`:
+```cmd
+<engine>\scripts\o3de.bat download --project-name RemoteProject
+```
+
+Example which downloads the latest version of a template named `RemoteTemplate` into `c:/o3de-templates/RemoteTemplate`:
+```cmd
+<engine>\scripts\o3de.bat download --template-name RemoteTemplate  --dest-path c:/o3de-templates
+```
```

Comment:
This looks like a windows example, use windows path separators instead
```suggestion
Example which downloads the latest version of a template named `RemoteTemplate` into `c:\o3de-templates\RemoteTemplate`:
```cmd
<engine>\scripts\o3de.bat download --template-name RemoteTemplate  --dest-path c:\o3de-templates
```
```